projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1441,5 +1453,6 @@ protected static function build_sprintf( $args ) {
  * @return string
  */
 function publicize_calypso_url() {
+	_deprecated_function( __METHOD__, 'jetpack-10.9', 'Publicize::publicize_connections_url' );
```

Comment:
Here you'll want to use `$$next-version$$` to ensure the version number is correct. Maybe we can tie this to the package instead since that is where we are deprecating it?

```suggestion
	_deprecated_function( __METHOD__, '$$next-version$$', 'Publicize::publicize_connections_url' );
```

Reply:
That's alright. Once we release a new version of the package, that string will be updated to the final version by a script.